### PR TITLE
update ticket-element.html

### PR DESCRIPTION
**Pull request by cliveb**
Recommend:

properties: {
          name: {
            type: String
          },

Was

properties: {
          type: {
            type: String
          },

### CODE REVIEW

File: src/elements/ticket-element.html [hunk 1]
```diff
@@ -155,7 +155,7 @@
         ],
 
         properties: {
-          type: {
+          name: {
```
Comment (ozasadnyy) [inline src/elements/ticket-element.html]: nope, we have `type` property 
```
<div class="side-content">[[type]]</div>
```

**Comment by ozasadnyy** [comment]
Thanks for PR, but everything is fine there. Check my code comment for details.

File: src/elements/ticket-element.html (hunk 1 above)
Comment (cliveb) [inline src/elements/ticket-element.html]: Ok. I didn't see type and type. I saw "name" and type. My bad.

    "elements": [
      {
        "name": "student-ticket",
        "price": 500,
        "currency": "₴",
        "starts": "student-ticket-starts",
        "ends": "student-ticket-ends",
        "info": "student-ticket-info",
        "soldOut": false
      }
    ]

Comment (ozasadnyy) [inline src/elements/ticket-element.html]: But you are right. It makes sense to be consistent everywhere

**Pull request closed without merging**